docs/network.mdx

```diff
@@ -2,41 +2,24 @@
 id: network
 title: Network Information
 description: Detailed information about chains, RPCs, block explorer and indexer
-sidebar_label: Network
+sidebar_label: Network Information
```

Comment:
Let's only keep the `description` field here. The sidebar label, title and id should be set in the sidebar.js.